api/graphql/schema.py

```diff
@@ -90,7 +107,7 @@ async def m(info: Info[GraphQLContext, 'Query']) -> list[str]:
 
 GraphQLEnum = strawberry.type(type('GraphQLEnum', (object,), enum_methods))
 
-GraphQLAnalysisStatus = strawberry.enum(AnalysisStatus)
+GraphQLAnalysisStatus = strawberry.enum(AnalysisStatus)  # type: ignore
```

Comment:
nit: does this mean the type is ignored?

Reply:
yeah this was a weird one, I didn't touch this line but it started erroring out in mypy, the ignore just silences the type error on this line but the type checking still works elsewhere